usmqe_tests/api/tendrl/README.rst

```diff
@@ -0,0 +1,3 @@
+There are 2 types of tests here:
```

Comment:
That's actually a good question. However in my first comment I just want to be sure that we count with such tests and I want to have some mention about them in the readme file.